rom/dev/tests/test_image_validation.rs

```diff
@@ -1438,6 +1438,7 @@ fn update_header(image_bundle: &mut ImageBundle) {
             &config,
```

Comment:
Integration tests seem to be missing

Reply:
Please add integration tests covering the following errors: IMAGE_VERIFIER_ERR_VENDOR_LMS_PUB_KEY_INDEX_MISMATCH
IMAGE_VERIFIER_ERR_VENDOR_LMS_VERIFY_FAILURE
IMAGE_VERIFIER_ERR_VENDOR_LMS_PUBKEY_INDEX_OUT_OF_BOUNDS
IMAGE_VERIFIER_ERR_VENDOR_LMS_SIGNATURE_INVALID
IMAGE_VERIFIER_ERR_UPDATE_RESET_VEN_LMS_PUB_KEY_INDEX_OUT_OF_BOUNDS
IMAGE_VERIFIER_ERR_OWNER_LMS_PUB_KEY_INDEX_MISMATCH
IMAGE_VERIFIER_ERR_OWNER_LMS_VERIFY_FAILURE
IMAGE_VERIFIER_ERR_OWNER_LMS_PUBKEY_INDEX_OUT_OF_BOUNDS
IMAGE_VERIFIER_ERR_OWNER_LMS_SIGNATURE_INVALID

Reply:
IMAGE_VERIFIER_ERR_VENDOR_LMS_VERIFY_FAILURE and IMAGE_VERIFIER_ERR_OWNER_LMS_VERIFY_FAILURE are never returned, but have integration tests for all paths now.

Reply:
Vishal and Vishal,
Can you please earmark 2 more fuses (32 bits each) for LMS key (vendor and owner) revocation also please?